glide.yaml

```diff
@@ -93,7 +93,6 @@ import:
 - package: k8s.io/client-go
   version: v2.0.0
 - package: github.com/gambol99/go-marathon
-  version: d672c6fbb499596869d95146a26e7d0746c06c54
```

Comment:
It's hard to explain, sorry if I will misspoke.

Currently when you add a dependency, you don't need to pin version to yml (as we did before)

Glide do some "random" update when some version are not fixed.
I don't know exactly why, but I know Glide love the past and not the present...
Some dependencies don't update when you add a dependency but not all...

To be "safe", some dependencies must be fixed. (IMHO in the yml all dependencies must be fixed on version or hash)

I know this is not prefect but I don't want to do again a +8 hours of hash analysis.